### PR TITLE
NEPT-1299: Removed db_select calls.

### PR DESCRIPTION
**Pull request by imanoleguskiza**
### Description

Removed db_select calls.

### Change log

- Fixed: Removed db_select calls.

### CODE REVIEW

**Comment by voidtek** [comment]
:+1:

File: profiles/common/modules/custom/multisite_config/lib/Drupal/locale/Config.php [hunk 1]
```diff
@@ -61,8 +61,12 @@ public function setLanguageNegotiation($language_negotiation, $type = LANGUAGE_T
    *    Language prefix.
    */
   public function setLanguagePrefix($language, $prefix) {
-    $arguments = array(':language' => $language, ':prefix' => $prefix);
-    db_query("UPDATE {languages} SET prefix = ':prefix' WHERE language = ':language'", $arguments)->execute();
+    db_update('languages')
+      ->fields(array(
+        'prefix' => $prefix,
+      ))
+      ->condition('language', $language, '=')
```
Comment (Fefaine) [inline profiles/common/modules/custom/multisite_config/lib/Drupal/locale/Config.php]: imo, the condition is more clear without the operator

> If no operator is specified, = is assumed. That means the most common case would be condition('myfield', $value), which results in a conditional fragment of myfield = :value, where :value will be replaced with $value when the query is run.

If you agree i will make the small change

File: profiles/common/modules/custom/multisite_og_navigation_tree/multisite_og_navigation_tree.install [hunk 2]
```diff
@@ -8,7 +8,12 @@
  * Implements hook_install().
  */
 function multisite_og_navigation_tree_install() {
-  db_query("UPDATE {system} SET weight = 5 WHERE name = 'multisite_og_navigation_tree'");
+  db_update('system')
+    ->fields(array(
+      'weight' => 5,
+    ))
+    ->condition('name', 'multisite_og_navigation_tree', '=')
```
Comment (Fefaine) [inline profiles/common/modules/custom/multisite_og_navigation_tree/multisite_og_navigation_tree.install]: idem

**Comment by Fefaine** [comment]
The change is just a detail, ready to merge.

**Pull request merged**